cli/cmd/create_service.go

```diff
@@ -40,9 +40,8 @@ Example:
 			return errors.New("required argument SERVICENAME not set")
 		}
 
-		if !keptnutils.ValidateKeptnEntityName(args[0]) {
-			errorMsg := "Service name contains upper case letter(s) or special character(s).\n"
-			return errors.New(errorMsg)
+		if !keptnutils.ValididateUnixDirectoryName(args[0]) {
+			return errors.New("Service name contains special character(s)")
```

Comment:
Maybe specify that the service name contains unsupported characters and add some information (e.g., via a hyperlink?) about those.

Reply:
I improved the error message.